iree/samples/vulkan/CMakeLists.txt

```diff
@@ -19,33 +19,32 @@ iree_bytecode_module(
     "simple_mul.mlir"
   CC_NAMESPACE
     "iree::samples::vulkan"
+  TRANSLATE_TOOL
+    iree_tools_iree-translate
+  TRANSLATION
+    "-iree-mlir-to-vm-bytecode-module"
+  PUBLIC
 )
 
-# Statically link against Vulkan.
-# Requires installing the Vulkan SDK from https://vulkan.lunarg.com/.
-include(FindVulkan)
-if(NOT Vulkan_FOUND)
-  message(FATAL_ERROR "Vulkan library was not found")
-endif()
-
```

Comment:
We need FindVulkan here! Maybe we should add it to the root CMakeLists.txt and encapsulate vulkan targets with a build switch.

Reply:
I think we're not building samples at all in the cmake build and I'm mostly relying on that to check correctness of the conversion. I think I'll just add a DO NOT EDIT to this file.

Reply:
I think we're not building samples at all in the cmake build and I'm mostly relying on that to check correctness of the conversion. I think I'll just add a DO NOT EDIT to this file.

Reply:
We build samples :)

https://github.com/google/iree/blob/26333e15fc14c5961551df9a98fb14c25dc7a763/CMakeLists.txt#L40

https://github.com/google/iree/blob/26333e15fc14c5961551df9a98fb14c25dc7a763/iree/samples/CMakeLists.txt#L17